datalab/datalab_session/tests/test_utils.py

```diff
@@ -0,0 +1,61 @@
+import pathlib as pl
+
+from django.test import TestCase
+
+from datalab.datalab_session.util import *
+from django.conf import settings
+
+# extending the TestCase class to include a custom assertions
+class TestCaseCustom(TestCase):
+    def assertIsFile(self, path):
+        if not pl.Path(path).resolve().is_file():
+            raise AssertionError("File does not exist: %s" % str(path))
+
+class S3UtilTestClass(TestCase):
+
+  def test_bucket(self):
+    # Test that the bucket is set to the correct value
+    self.assertEqual(settings.DATALAB_OPERATION_BUCKET, 'datalab-operation-output-lco-global')
```

Comment:
So this is not a good test, since the whole point of having that as a setting and env variable is that it can point to different buckets without changing code. You never want to test that the value of a env variable is something specific.